src/bluehawk/io/MockFs.ts

```diff
@@ -0,0 +1,145 @@
+import { strict as assert } from "assert";
```

Comment:
Any reason you didn't choose something like https://github.com/streamich/memfs ?

Reply:
Fun! But really, I searched up "mock fs" and what I found wasn't what I was looking for. This looks significantly better.